### PR TITLE
Create AWS secrets to hold Akita credentials

### PR DESCRIPTION
**Pull request by mgritter**
...if they do not already exist.

Check that credentials are present and project exists.

### CODE REVIEW

File: cmd/internal/ecs/add.go [hunk 1]
```diff
@@ -571,10 +589,38 @@ func getServiceState(wf *AddWorkflow) (nextState optionals.Optional[AddWorkflowS
 	return awf_next(confirmState)
```
Comment (liujed) [inline cmd/internal/ecs/add.go:589]:
```suggestion
	return awf_next(getSecretState)
```

File: cmd/internal/ecs/add.go [hunk 2]
```diff
@@ -122,6 +137,9 @@ func RunAddWorkflow() error {
 //    |-getService
 //           |
 //           V
+//     checkForSecret
```
Comment (liujed) [inline cmd/internal/ecs/add.go:140]:
```suggestion
//    |-getService
//    |      |
//    |      V
//    |- getSecret
```

File: cmd/internal/ecs/add.go [hunk 3]
```diff
@@ -664,5 +715,42 @@ func fillFromFlags(wf *AddWorkflow) (nextState optionals.Optional[AddWorkflowSta
 }
 
 func addSecretState(wf *AddWorkflow) (nextState optionals.Optional[AddWorkflowState], err error) {
-	return awf_error(errors.New("Unimplemented!"))
+	reportStep("Add AWS Secret")
+
+	key, secret := cfg.GetAPIKeyAndSecret()
```
Comment (liujed) [inline cmd/internal/ecs/add.go:720]: Would be nice if we requested a new key and secret from the back end for use in ECS. Not a blocker, though.

**Review by liujed — APPROVED** [review]

File: cmd/internal/ecs/add.go (hunk 1 above)
Comment (mgritter) [reply cmd/internal/ecs/add.go:589]: I am very confused as to how this worked when I tested it.

... oh, because they both default to false.

**Pull request merged**